DigitalOffice/Services/FileService/Mappers/FileMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using System;
+
+namespace FileService.Mappers
+{
+    public class FileMapper : IMapper<DbFile, FileRequest>, IMapper<FileRequest, DbFile>
+    {
+        public FileRequest Map(DbFile dbFile)
+        {
+            return new FileRequest()
+            {
+                Id = dbFile.Id,
+                Content = Convert.ToBase64String(dbFile.Content),
+                Extension = dbFile.Extension,
+                Name = dbFile.Name
+            };
+        }
+        public DbFile Map(FileRequest file)
```

Comment:
I think, we should add here check for null

Reply:
What kind of check? To throw an exception?
It will already do that for us.
We should add null check when we have an ExceptionHandler in our Middleware. And even after that an exception will be thrown and be caught in the middleware, so checks on the mapper level are unnecessary.

Reply:
As far as I know, it is useful to provide the most specific information with an exception throwing. For example, in this case, if the argument is null you should throw ArgumentNullException, BUT your code will throw NullReferenceException. Obviously, the exceptions vary and ArgumentNullException provides the most accurate information. It is in our interests that the ExceptionHandler receives the most accurate information.

Reply:
Interesting point. I'll give it some thought

Reply:
Fixed, added null checks.
Great idea, Mikhail.